macros/psng_manual_change.ngc

```diff
@@ -17,11 +19,12 @@ O100     return [999]
 O100 endif
 
 ;first go up
+F#<_ini[TOOLSENSOR]RAPID_SPEED>
 G90
-G53 G0 Z[#<_ini[AXIS_Z]MAX_LIMIT>-0.1]
+G53 G1 Z[#<_ini[AXIS_Z]MAX_LIMIT>-1]
```

Comment:
Why change from -0.1 to -1? The goal is is to get as high as possible to reduce the risk of making contact with the part.

Reply:
you are right it's only because dislike to be too close to the sensor, but this is ok with 0.1